### ISSUE TITLE
Tobit Regression Assignment (80/100)

### ISSUE THREAD

**Issue by joelowj** [joelowj/Machine-Learning-and-Reinforcement-Learning-in-Finance#1]
Current submissions

Tobit Model Weights                                         20/20
Tobit Model: loss, standard error, L1 error .     20/20
Linear Regression with non-linear data          20/20
Linear Regression Model Score                       20/20
Neural Network with Non-Linear Data              20/20

My implementation for Neural Network with Non-Linear Data did not clear the grader. 
Feel free to start a discussion if you managed to clear the grader or have an even better implementation.

**Comment by joelowj** [comment]
89a605c74889d1202dac1794444fbec3d42aaab4, Tobit Model Weights & Tobit Model: loss, standard error, L1 error has been cleared.

It seems that the r square coefficient approach for Neural Network with Non-Linear Data did not clear the grader.

**Comment by joelowj** [comment]
36468806b9fc30a4cf39cc0e6517fd2986c0ad59, Neural Network with Non-Linear Data has been cleared. I have changed the approached from using 

u = tf.reduce_sum(tf.square(output - y))
v = tf.reduce_sum(tf.square(tf.reduce_mean(y) - y))
r_square_coefficient = 1 - (u / v)

to just evaluation the test accuracy on loss which was defined to be 
loss = tf.reduce_mean(tf.square(output - y)). The solution obtained was acc_test = 0.041695457.
